portal/app/Http/Kernel.php

```diff
@@ -16,7 +16,7 @@ class Kernel extends HttpKernel
     protected $middleware = [
         \App\Http\Middleware\CheckForMaintenanceMode::class,
         \Illuminate\Foundation\Http\Middleware\ValidatePostSize::class,
-        \App\Http\Middleware\Authenticate::class,
+        //\App\Http\Middleware\Authenticate::class,
```

Comment:
Is this intentional?

Reply:
Yes, authentication is disabled now.

Reply:
Can we make authentication optional, e.g.: a login link?